server/views/components/promo/index.njk

```diff
@@ -3,64 +3,76 @@
           {% icon 'actions/gallery-link' %}
-        {% elif meta.type === 'podcast' %}
+        {% elif promo.meta.type === 'podcast' %}
           {% icon 'actions/volume'  %}
-        {% elif meta.type === 'video' %}
+        {% elif promo.meta.type === 'video' %}
             {% icon 'actions/play' %}
         {% endif %}
-        {% if meta.type === 'gallery' and meta.length %}
-          <span class="promo__length"><span class="promo__sr">Gallery with</span> {{ meta.length }} <span class="promo__sr">item{{ 's' if meta.length > 1 }}</span></span>
+        {% if promo.meta.type === 'gallery' and promo.meta.length %}
+          <span class="promo__length"><span class="promo__sr">Gallery with</span> {{ promo.meta.length }} <span class="promo__sr">item{{ 's' if promo.meta.length > 1 }}</span></span>
         {% endif %}
-        {% if meta.type === 'podcast' or meta.type === 'video' and meta.length %}
-          <span class="promo__sr">{% if meta.type === 'podcast' %}Podcast{%else%}Video{% endif %} lasting</span>
-          {{ meta.length }}
+        {% if promo.meta.type === 'podcast' or promo.meta.type === 'video' and promo.meta.length %}
+          <span class="promo__sr">{% if promo.meta.type === 'podcast' %}Podcast{%else%}Video{% endif %} lasting</span>
+          {{ promo.meta.length }}
         {% endif %}
       </div>
     {% endif %}
   </div>
-  <header class="promo__description">
-    {% if meta.type %}
-      <span class="promo__type" aria-hidden="true">{{ meta.type }}</span>
-    {% endif %}
-      <{{ headingLevel or 'h2'}} class="promo__title">
-        {{ title }}
-      </{{ headingLevel or 'h2'}}>
-    {% if meta.date %}
-      <span class="promo__date">{{ meta.date }}</span>
-    {% endif %}
-    {% if copy %}
-      <span  class="promo__copy">{{ copy }}</span>
-    {% endif %}
-  </header>
+
+  {% if promo.modifiers and promo.modifiers | contains('standalone') %}
+    <div class="row row--cream">
+      <div class="container">
+        <div class="grid">
+          <div class="{{ {l:10, shiftL:1, m:6, shiftM:1, s:4} | gridClasses }}">
+          {% endif %}
+            <header class="promo__description">
+              <div class="promo__heading">
+                {% if promo.meta.type %}
+                  <span class="promo__type" aria-hidden="true">{{ promo.meta.type }}</span>
+                {% endif %}
+                  <{{ headingLevel or 'h2'}} class="promo__title">
+                    {{ promo.article.headline }}
+                  </{{ headingLevel or 'h2'}}>
+                {% if promo.meta.date %}
+                  <span class="promo__date">{{ promo.meta.date }}</span>
+                {% endif %}
+              </div>
+              {% if promo.copy %}
+                <span  class="promo__copy">{{ promo.copy | striptags | safe | truncate(100) }}</span>
+              {% endif %}
+            </header>
+          {% if promo.modifiers and promo.modifiers | contains('standalone') %}
```

Comment:
`if`s embedded like this are so prone to error.
Is there a way to use `block`s here instead?